cmd/crc/cmd/daemon_linux.go

```diff
@@ -9,9 +9,9 @@ import (
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/containers/gvisor-tap-vsock/pkg/transport"
 
+	"github.com/cfergeau/vsock"
```

Comment:
fwiw, it probably makes more sense to have this commit first in the PR.

Reply:
If there is no strong opinion then I would have it as it is.